Merge main to live 11/17/23 (#52)
MicrosoftDocs/community-content: community-content/content/must-use-ai-features-in-power-bi.md

---
title: Must-Use AI Features in Power BI 
description: This article is your companion to becoming effective business data analyst using Microsoft Power BI and leveraging on its AI capabilities.

> author: jehchow #Required; a Microsoft employee's GitHub user alias, with correct capitalization; don't change.

Author: ```suggestion
author: theoyinbooke  #Required; a Microsoft employee's GitHub user alias, with correct capitalization; don't change.
```

> author: jehchow #Required; a Microsoft employee's GitHub user alias, with correct capitalization; don't change.

Author: ```suggestion
author: cahublou #Required; a Microsoft employee's GitHub user alias, with correct capitalization; don't change.
```

> ms.author: shchow #Required; a Microsoft employee's alias; don't change. 

Author: ```suggestion
ms.author: cahublou #Required; a Microsoft employee's alias; don't change. 
```

> ms.service: power-bi #Required; service per approved list. service slug assigned to your service by ACOM.

Author: ```suggestion
ms.service: powerbi #Required; service per approved list. service slug assigned to your service by ACOM.
```

## The Pareto Principle for Data Analysts
I believe Business Intelligence is shifting to Decision Intelligence. Decision Intelligence is where much value can be realized from your Data Analysis effort. This is where businesses get to act on your recommendation. 
I tag this "The Pareto Principle for Data Analysts" because according to the general knowledge of Pareto principle, it emphasizes controlling the 20% that has the 80% effect. Isn't that incredible? - If you exert 20% effort to get 80% impact.
Surprisingly, it is easier to achieve this with Power BI. The AI features here help you do the little things that make the most impact.

## You'll Never Work Alone
You read that right? With the AI Features in Power BI, you'll never work alone. Imagine how much value you can create for the business by leveraging the AI features in this learning path, I am already excited on your behalf and I can't wait to see you get transformed on this skilling journey.

## Time to Get Started

> I have curated the contents you need to learn how to use these AI features and practice them below, click the link or the image to get started.
> [Click here to start learning](https://learn.microsoft.com/collections/1kp7t5go8ejmx5)
> 

Author: ```suggestion
I have curated the contents you need to learn how to use these AI features and practice them. [Click here to start learning](https://learn.microsoft.com/collections/1kp7t5go8ejmx5).

```
